src/styles/julabat.js

```diff
@@ -123,28 +128,37 @@ export default class JuLabatStyle extends Style {
     )
 
     // Glow effect
-    P5.blendMode(P5.SCREEN)
-    if (isBorder) P5.fill(darkBgColor)
-    else P5.fill(h, 100, 80)
-    this.setBlur(40)
-    P5.quad(
-      p0.x * this._s, p0.y * this._s,
-      p1.x * this._s, p1.y * this._s,
-      p2.x * this._s, p2.y * this._s,
-      p3.x * this._s, p3.y * this._s
-    )
+    if (this.tileGlow) {
+      P5.blendMode(P5.SCREEN)
+      if (isBorder) P5.fill(darkBgColor)
+      else P5.fill(h, 100, 80)
+      this.setBlur(40)
+      P5.quad(
+        p0.x * this._s, p0.y * this._s,
+        p1.x * this._s, p1.y * this._s,
+        p2.x * this._s, p2.y * this._s,
+        p3.x * this._s, p3.y * this._s
+      )
+    }
 
     // Reset effects
-    this.setBlur(0)
+    this.removeBlur()
     P5.blendMode(P5.BLEND)
   }
 
-  afterDraw () {}
+  afterDraw () {
+    const P5 = this._p5
+    if (P5.frameCount > 1) P5.noLoop()
```

Comment:
noLoop() is already called in index.js.

Reply:
It's the only trick I found to resolve my resizing issue. I didn't found out exactly what caused the problem but I solved it by enabling the script to loop for just one frame.

Reply:
Ok everything is fixed now thanks to @solarisedesign